src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -605,7 +605,7 @@ private function createResponsiveSizer(
         $style = 'padding-top:%s%%'
     ) {
         $padding       = $height->getNumeral() / $width->getNumeral() * 100;
-        $paddingString = (string) round($padding, 4);
+        $paddingString = preg_replace('/\.?0*$/', '', sprintf('%.4F', $padding));
```

Comment:
The PHP community chimed in here: https://twitter.com/schlessera/status/1333330595179532288

The regex can be replaced by a faster 2x `rtrim()` operation instead.
Also, the `sprintf()` does not round the number, so to keep the previous behavior intact, we need to `round()`  first.

```suggestion
        $paddingString = rtrim(rtrim(sprintf('%.4F', round($padding, 4)), '0'), '.');
```

Reply:
you should even be able to only use 1 `rtrim` with `'0.'` as the second parameter...

Reply:
That would actually strip significant zeros on the left side of the period as well, i.e. `100.0000` would be trimmed into `1`.

Reply:
Dammit. You're right!